v23.1/admission-control.md

```diff
@@ -5,15 +5,15 @@ toc: true
 docs_area: develop
 ---
 
-CockroachDB supports an admission control system to maintain cluster performance and availability when some nodes experience high load. When admission control is enabled, CockroachDB sorts request and response operations into work queues by priority, giving preference to higher priority operations. Internal operations critical to node health, like node liveness heartbeats, are high priority. The admission control system also prioritizes transactions that hold locks, to reduce contention by releasing locks in a timely manner.
+CockroachDB supports an admission control system to maintain cluster performance and availability when some nodes experience high load. When admission control is enabled, CockroachDB sorts request and response operations into work queues by priority, giving preference to higher priority operations. Internal operations critical to node health, like [node liveness heartbeats](cluster-setup-troubleshooting.html#node-liveness-issues), are high priority. The admission control system also prioritizes transactions that hold [locks](architecture/transaction-layer.html#writing), to reduce [contention](performance-best-practices-overview.html#transaction-contention) by releasing [locks](crdb-internal.html#cluster_locks) in a timely manner.
```

Comment:
Are the three separate links here at all confusing?

Reply:
Yes probably.  I've updated this to make only the first mention of "locks" a link to https://www.cockroachlabs.com/docs/v22.1/crdb-internal.html#cluster_locks.  My thinking for linking to that page is that it may be more practical for most users, and it itself links to the architecture docs on locks at https://www.cockroachlabs.com/docs/v22.1/architecture/transaction-layer#concurrency-control